config/initializers/sso.rb

```diff
@@ -0,0 +1,2 @@
+Rails.configuration.sso_admin_org = ENV.
+fetch('sso_admin_org', 'pvb.digital.headquarters.moj')
```

Comment:
Honestly I'd be happy just hard-coding this in the code – it will change that rarely.